### PR TITLE
Changelog Addition

### PR DESCRIPTION
**Pull request by jvanderhoof**
Changelog entry for previous change.

@apotterri

### CODE REVIEW

**Comment by apotterri** [comment]
I put comments here https://github.com/conjurinc/api-ruby/pull/104 before you closed it ;)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # v5.0.0
```
Comment (apotterri) [inline CHANGELOG.md]: This should match what's in `version.rb`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
+* Removed hard dependency on older version of `rest-client` gem.
+
 WIP
```
Comment (apotterri) [inline CHANGELOG.md]: Do we want this to be here, still?

**Comment by jvanderhoof** [comment]
@apotterri: both of your comments were addressed.  Thanks for catching!

**Pull request merged**